app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -308,6 +308,11 @@ fun CreateActivityScreen(
                 val calendar = GregorianCalendar()
                 val parts = dueDate.split("/")
                 if (parts.size == 3 && timeFormat.size == 2 && durationFormat.size == 2) {
```

Comment:
it's good that you thought about adding the user the list of participants once you enroll in an activity.